### PR TITLE
Readme refactor

### PR DESCRIPTION
**Pull request by JarateKing**
general additions and reformatting of the readme

### CODE REVIEW

**Comment by JarateKing** [comment]
I flubbed up branching a bit, and this branch is based off the branch in #5 -- please handle that PR before this one!

**Comment by getchoo** [comment]
>this branch is based off the branch in https://github.com/treacherousfiend/LightmapUtil/pull/5

@JarateKing `git rebase`

image

File: README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ A simple command line utility to tell you if your lightmaps are too high resolut
 
 ## Usage
 
+The program can be run with `LightmapUtil.exe filename.bsp`. This generates `filename-LightmapUtilLog.txt` that provides you with details about lightmapping on that map file.
```
Comment (getchoo) [inline README.md:7]: you should add the linux equivalent here `lightmaputil filename.bsp` 😀

Comment (treacherousfiend) [reply README.md:7]: you don't even need the .exe on windows, but its really not a big deal rn, linux builds aren't provided (yet)

**Pull request merged**